data/phi0631/phi001/phi0631.phi001.perseus-eng2.xml

```diff
@@ -207,11 +207,11 @@ So Silius Italicus, xv. 84:
 <p>By these two virtues, intrepidity in war, and equity in peace, they maintained themselves and their state. Of their exercise of which virtues, I consider these as the greatest proofs; that, in war, punishment was oftener inflicted on those who attacked an enemy contrary to orders, and who, when commanded to retreat, retired too slowly from the contest, than on those who had dared to desert their standards, or, when pressed by the enemy,<note anchored="true" place="foot">IX. Pressed by the enemy] <quote xml:lang="lat">Pulsi.</quote> In the words <foreign xml:lang="lat">pulsi loco cedere ausi erant, loco</foreign> is to be joined, as Dietsch observes, with <foreign xml:lang="lat">cedere,</foreign> not, as Kritzius puts it, with <foreign xml:lang="lat">pulsi.</foreign> "To retreat," adds Dietsch, " is disgraceful only to those <foreign xml:lang="lat">qui ab hostibus se pelli patiantur,</foreign> who suffer themselves to be repulsed by the enemy."</note> to abandon their posts; and that, in peace, they governed more by conferring benefits than by exciting terror, and, when they received an injury, chose rather to pardon than to revenge it.</p></div>
 <div type="textpart" subtype="chapter" n="10">
 <p> But when, by perseverance and integrity, the republic had increased its power; when mighty princes had been vanquished in war;<note anchored="true" place="foot">X. When mighty princes had been vanquished in war] Perses, Antiochus, Mithridates, Tigranes, and others.</note> when barbarous tribes and populous states had been reduced to subjection; when <placeName key="tgn,7016143">Carthage</placeName>, the rival of <placeName key="tgn,7013962">Rome</placeName>'s dominion, had been utterly destroyed, and sea and land lay <pb n="15"/>every where open to her sway, Fortune then began to exercise her tyranny, and to introduce universal innovation. To those who had easily endured toils, dangers, and doubtful and difficult circumstances, ease and wealth, the objects of desire to others, became a burden and a trouble. At first the love of money, and then that of power, began to prevail, and these became, as it were, the sources of every evil. For avarice subverted honesty, integrity, and other honorable principles, and, in their stead, inculcated pride, inhumanity, contempt of religion, and general venality. Ambition prompted many to become deceitful; to keep one thing concealed in the breast, and another ready on the tongue;<note anchored="true" place="foot">To keep one thing concealed in the breast, and another ready on the tongue] <quote xml:lang="lat">Aliud clausum in pectore, aliud in linguâ promptum, habere.</quote>
-<quote><foreign xml:lang="grc">Ἐχθρὸσ γάρ μοι κεῖνοσ ὁμῶσ Ἀΐδαο πύλῃσιν</foreign><lb/>
-<foreign xml:lang="grc">Ὃσ χ</foreign><lb/>
+<quote><foreign xml:lang="grc"><l>Ἐχθρὸσ γάρ μοι κεῖνοσ ὁμῶσ Ἀΐδαο πύλῃσιν</foreign></l>
```

Comment:
If rendered in print as a block quote, the convention is `<quote rend="align(block quote)">` with lines indented if appropriate (if there is further indentation)
`<l\>` or wrapping `<l>`  is fine within that.
Much of this will be reviewed as part of the Latin transition, where I will check for inconsistencies and inaccuracies in this markup. Very little of the Latin work is up to current best practices.

Reply:
This looked odd to me but I can see where the elision is not clearly shown in the print.
It looks like there is no terminal sigma in this Greek and this second line should read `ὅς χʼ `
Issues with Greek presentation are something that needs to be reviewed. There are some common errors with the way some of this was converted. Again, more of a corpus-wide problem than one particular to any text.

Reply:
the text actually has a full line after `Ὃσ χ` that is omitted in the digitization but not marked anywhere.

I have done my best to add it myself and get the accents correct, but my familiarity with Greek is extremely low so please take a careful look.

My transcription
> ὅς χ’ ἕτερον μὲν κεύθει ἐνὶ φρεσὶν, ἄλλο δὲ βάζει

Link to the page in question:
https://archive.org/details/sallustflorusvel00salliala/page/14/mode/2up